generated/nidcpower/session.py

```diff
@@ -2378,6 +2378,55 @@ def fetch_multiple(self, count, timeout=datetime.timedelta(seconds=1.0)):
         the *NI DC Power Supplies and SMUs Help* for more information about
         configuring this method.
 
+        Note: This method is not supported on all devices. Refer to `Supported Methods by Device <REPLACE_DRIVER_SPECIFIC_URL_2(nidcpowercref.chm, supportedfunctions)>`__ for more information about supported devices.
+
+        Tip:
+        This method requires repeated capabilities (usually channels). If called directly on the
+        nidcpower.Session object, then the method will use all repeated capabilities in the session.
+        You can specify a subset of repeated capabilities using the Python index notation on an
+        nidcpower.Session instance, and calling this method on the result.:
+
+            session.channels['0,1'].fetch_multiple(count, timeout=datetime.timedelta(seconds=1.0))
+
+        Args:
+            count (int): Specifies the number of measurements to fetch.
+
+            timeout (datetime.timedelta): Specifies the maximum time allowed for this method to complete, in seconds. If the method does not complete within this time interval, NI-DCPower returns an error.
+
+                Note: When setting the timeout interval, ensure you take into account any triggers so that the timeout interval is long enough for your application.
+
+
+        Returns:
+            measurements (list of float): Returns an array of named tuples.
+
+        '''
+        import collections
+        Measurements = collections.namedtuple('Measurements', ['voltage', 'current', 'in_compliance'])
```

Comment:
I think the name of the tuple should be singular `Measurement`, right?

Reply:
It contains the voltage and the current measurement, but I'm pretty sure they are considered one thing. The count is 1 for each of these.

Reply:
We do consider them one thing, and we try to refer to the constituents as "measurement components."